1.5/Source/PawnStorages/PawnStorages/Farm/Interfaces/IBreederParent.cs

```diff
@@ -6,8 +6,13 @@ namespace PawnStorages.Farm.Interfaces
     public interface IBreederParent : IActive
     {
         public List<Pawn> BreedablePawns { get; }
+        public List<Pawn> AllPawns { get; }
         public int TickInterval { get; }
 
         public void Notify_PawnBorn(Pawn newPawn);
+
+        public void Release(Pawn pawn);
```

Comment:
if you call this ReleasePawn you avoid the earlier duplication

Reply:
Yup, done.